include/blosc2.h

```diff
@@ -1819,6 +1825,19 @@ BLOSC_EXPORT blosc2_schunk* blosc2_schunk_open(const char* urlpath);
  */
 BLOSC_EXPORT blosc2_schunk* blosc2_schunk_open_offset(const char* urlpath, int64_t offset);
 
+/**
+ * @brief Open an existing super-chunk that is on-disk (frame) in memory-mapped mode. No in-memory copy is made.
+ *
+ * @param urlpath The file name.
+ *
+ * @param offset The frame offset.
+ * 
+ * @param mmap_mode The mode for the memmory mapping similar to Numpy's np.memmap (https://numpy.org/doc/stable/reference/generated/numpy.memmap.html). Set to r if the super-chunk should only be read. See #blosc2_stdio_mmap struct for details.
```

Comment:
Line is too long.  See updated developer guidelines: https://github.com/Blosc/c-blosc2/blob/main/DEVELOPING-GUIDE.rst